less/index.less

```diff
@@ -52,8 +52,139 @@ table {
 * {
     box-sizing: border-box;
 }
+body {
+	height: 100%;
+	// border: 1px solid black;
+  }
+@main-bg:        white;
+@font-stack: 'Roboto', Arial, sans-serif;
+.container{
+	// height: 100%;
+	width: 1024px;
+  	margin: 0 auto;
+  	background: @main-bg;
+  	font-family: @font-stack;	
+}
+.custom-flex( @justify, @align ){
+	display: flex;
+	justify-content: @justify;
+	align-item:@align;
+  }
+  header nav{
+	.custom-flex(flex-end, space-between);
+	padding: 0%;
+	a{text-decoration: none;}
+	
+	 a:hover {
+		text-decoration: underline;
+	  }
+  }
+  header nav a{
+	  padding: 0% 2%;
+	  margin: 1% 0% 0 0;
+  }
+  
+  .top-section img{
+	  margin:3% 0% 1% 3%;
+  }
+  .hfont(){
+	  font-size: 200%;
+  }
+  .pfont(){
+	  font-size: 100%;
+  }
+  .inmarleft(){
+	  margin-left:3%;
+  }
+  .top-section{
+	background: pink;
+  // .custom-flex(inline-block, center)
+	  display: inline-block;
```

Comment:
You have a mixture of flex box and inline-block. I would stick with one or the other. So either all flexbox or all inline-block and block.